src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -2313,7 +2313,7 @@ describe('ReactDOMComponent', () => {
 
         expectDev(console.error.calls.count()).toBe(1);
         expectDev(console.error.calls.argsFor(0)[0]).toContain(
-          'Warning: Invalid DOM property `x-height`. Did you mean `xHeight`',
+          'Warning: Unknown DOM property `x-height`. Did you mean `xHeight`',
         );
       });
 
```

Comment:
`Invalid` is my fault. I changed it to `Invalid` because we _do_ know the attribute, and we know that it is being used incorrectly. Having said that, I have no real preference.

Reply:
This seems okay to me but shouldn't we do this consistently? We don't use `Invalid` for event handlers.

Reply:
Yeah. I'd even throw out that something like `Unsupported event listener` is more appropriate. I don't think the warning is made better by the inconsistency, let's just go with `unknown`.